src/sqlfluff/core/rules/std/L030.py

```diff
@@ -38,4 +39,12 @@ class Rule_L030(Rule_L010):
 
     """
 
-    _target_elems: List[Tuple[str, str]] = [("name", "function_name")]
+    _target_elems: List[Tuple[str, str]] = [("type", "function_name")]
+
+    def _get_fix(self, segment, fixed_raw):
```

Comment:
Likewise I think this function needs some explanation in the docstring. It's not obvious why we need to to this from the code.